portal/eproms/templates/eproms/website_consent_script.html

```diff
@@ -2,6 +2,10 @@
 {% block main %}
 	{%- from "initial_queries_macros.html" import paperWebsiteConsentScript, interviewAssistedWebConsentScript -%}
 	<div id="termsContainer" class="website-consent-script">
+		<input id="wcsPatientId" value="{{patient_id}}" type="hidden" />
+		<input id="wcsEntryMethod" value="{{entry_method}}"  type="hidden" />
```

Comment:
Are you able to get these values from the URL as querystring parameters?

Reply:
@ivan-c Ivan, that makes sense.  I made the fixes.  Let me know if you see issue with my fixes.